core/internal/filetransfer/file_transfer_default.go

```diff
@@ -39,6 +39,11 @@ func NewDefaultFileTransfer(
 	return fileTransfer
 }
 
+// CanHandle returns true if DefaultFileTransfer can upload/download the task
+func (ft *DefaultFileTransfer) CanHandle(task *Task) bool {
```

Comment:
I believe this will become useful in the future? Would be nice to explain a bit more in the comments why it always returns True for now (maybe later when we support multiple ft implementations).